srcts/src/components/sidebar.ts

```diff
@@ -528,7 +528,5 @@ class SidebarInputBinding extends InputBinding {
 }
 
 registerBinding(SidebarInputBinding, "sidebar");
```

Comment:
I'd be curious to know how `registerBinding()` handles this (i.e., does the last one win)? Seems like it'd be wise to match whatever it does?

Reply:
Would we want adding `bslib::value_box()` to a Quarto dashboard to use Quarto's components -- that they test against and expect to work -- or to use bslib's component js instead?

Reply:
Ah, good point, makes sense for the 1st one to win then. But also, if the last one wins for `registerBinding()`, wouldn't the binding be using the wrong `Sidebar` (i.e., we should maybe update `registerBinding()` to do something similar to `registerBslibGlobal()`)?

Reply:
`registerBinding()` is basically a safe call to `BindingsRegistry.register()` in shiny. "registering" a binding just adds it to the end of an array. The `.getBindings()` method breaks ties using the `priority` propery, followed by most-recently-registered.

Anyway, you're pointing out that this isn't an unambiguous or complete fix, and I agree. This PR at least gets us out of the position of breaking Dashboards. We'll have to do some thinking in the future